src/chains/models.py

```diff
@@ -24,6 +24,7 @@ class Chain(models.Model):
     )  # A lower number will indicate more relevance
     name = models.CharField(verbose_name="Chain name", max_length=255)
     rpc_uri = models.URLField()
+    safe_apps_rpc_uri = models.URLField()
```

Comment:
So this would be a mandatory field right?

Reply:
I'm not sure to be honest. On the frontend, it's a mandatory field. Maybe we can make it optional and use the main rpc URL as a fallback

Reply:
One idea could be to make it optional and when serialising it returns the `rpc_uri` if this is not set. What do you think? (same for authentication).

But if `safe_apps_rpc_uri` or `safe_apps_rpc_authentication` is set then both should exist in the model.

This can be done in a following PR if we agree to this